This patches the LibreSSL-portable build system to support our extra
architectures and ABIs.

--- a/configure.ac
+++ b/configure.ac
@@ -118,6 +118,12 @@ AM_CONDITIONAL([HOST_ASM_ELF_ARM],
     [test "x$HOST_ABI" = "xelf" -a "$host_cpu" = "arm" -a "x$enable_asm" != "xno"])
 AM_CONDITIONAL([HOST_ASM_ELF_X86_64],
     [test "x$HOST_ABI" = "xelf" -a "$host_cpu" = "x86_64" -a "x$enable_asm" != "xno"])
+AM_CONDITIONAL([HOST_ASM_ELF_PPC64LE],
+    [test "x$HOST_ABI" = "xelf" -a "$host_cpu" = "powerpc64le" -a "x$enable_asm" != "xno"])
+AM_CONDITIONAL([HOST_ASM_ELF_PPC64V2],
+    [test "x$HOST_ABI" = "xelf" -a "$host_cpu" = "powerpc64" -a "x$enable_asm" != "xno"])
+AM_CONDITIONAL([HOST_ASM_ELF_PPC],
+    [test "x$HOST_ABI" = "xelf" -a "$host_cpu" = "powerpc" -a "x$enable_asm" != "xno"])
 AM_CONDITIONAL([HOST_ASM_MACOSX_X86_64],
     [test "x$HOST_ABI" = "xmacosx" -a "$host_cpu" = "x86_64" -a "x$enable_asm" != "xno"])
 AM_CONDITIONAL([HOST_ASM_MASM_X86_64],
diff --git a/crypto/Makefile.am b/crypto/Makefile.am
index b897c00..512e4c7 100644
--- a/crypto/Makefile.am
+++ b/crypto/Makefile.am
@@ -212,12 +212,18 @@ EXTRA_libcrypto_la_SOURCES =
 
 include Makefile.am.elf-arm
 include Makefile.am.elf-x86_64
+include Makefile.am.elf-ppc64le
+include Makefile.am.elf-ppc64v2
+include Makefile.am.elf-ppc
 include Makefile.am.macosx-x86_64
 include Makefile.am.masm-x86_64
 include Makefile.am.mingw64-x86_64
 
 if !HOST_ASM_ELF_ARM
 if !HOST_ASM_ELF_X86_64
+if !HOST_ASM_ELF_PPC64LE
+if !HOST_ASM_ELF_PPC64V2
+if !HOST_ASM_ELF_PPC
 if !HOST_ASM_MACOSX_X86_64
 if !HOST_ASM_MASM_X86_64
 if !HOST_ASM_MINGW64_X86_64
@@ -233,6 +239,9 @@ endif
 endif
 endif
 endif
+endif
+endif
+endif
 
 libcrypto_la_SOURCES += cpt_err.c
 libcrypto_la_SOURCES += cryptlib.c
@@ -256,6 +265,7 @@ noinst_HEADERS += cryptlib.h
 noinst_HEADERS += md32_common.h
 noinst_HEADERS += o_time.h
 noinst_HEADERS += x86_arch.h
+noinst_HEADERS += ppc_arch.h
 
 # aes
 libcrypto_la_SOURCES += aes/aes_cfb.c
